src/manager.js

```diff
@@ -75,9 +75,7 @@ AdManager.prototype.handleWindowResize = function() {
 AdManager.prototype.initGoogleTag = function() {
   var adManager = this;
 
-  this.googletag.pubads().enableSingleRequest();
   this.googletag.pubads().disableInitialLoad();
-  this.googletag.pubads().collapseEmptyDivs(true);
```

Comment:
Have to remove otherwise GPT sets the display of the ad slot containers to `display: none` and then the bulbs-dfp object is unable to calculate positioning of the bounding client rect.  If `display: none`, all positions set to 0.